$NetBSD: patch-Makefile,v 1.5 2015/11/03 10:48:55 jperkin Exp $

* Fix build on systems lacking strverscmp.
* Use CFLAGS that were passed in
* Fix DESTDIR.
* Fix PKGMANDIR.

--- Makefile.orig	2014-04-18 17:35:39.000000000 +0000
+++ Makefile
@@ -24,20 +24,20 @@ VERSION=1.7.0
 TREE_DEST=tree
 BINDIR=${prefix}/bin
 MAN=tree.1
-MANDIR=${prefix}/man/man1
+MANDIR=${prefix}/${MANSUBDIR}/man1
 OBJS=tree.o unix.o html.o xml.o json.o hash.o color.o
 
 # Uncomment options below for your particular OS:
 
 # Linux defaults:
-CFLAGS=-ggdb -Wall -DLINUX -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64
+#CFLAGS=-ggdb -Wall -DLINUX -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64
 #CFLAGS=-O4 -Wall  -DLINUX -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64
 #LDFLAGS=-s
 
 # Uncomment for FreeBSD:
 #CFLAGS=-O2 -Wall -fomit-frame-pointer
 #LDFLAGS=-s
-#OBJS+=strverscmp.o
+OBJS+=strverscmp.o
 
 # Uncomment for Solaris:
 #CC=cc
@@ -95,12 +95,12 @@ clean:
 	rm -f *~
 
 install: tree
-	install -d $(BINDIR)
-	install -d $(MANDIR)
+	install -d $(DESTDIR)$(BINDIR)
+	install -d $(DESTDIR)$(MANDIR)
 	if [ -e $(TREE_DEST) ]; then \
-		install $(TREE_DEST) $(BINDIR)/$(TREE_DEST); \
+		install $(TREE_DEST) $(DESTDIR)$(BINDIR)/$(TREE_DEST); \
 	fi
-	install doc/$(MAN) $(MANDIR)/$(MAN)
+	install doc/$(MAN) $(DESTDIR)$(MANDIR)/$(MAN)
 
 distclean:
 	if [ -f tree.o ]; then rm *.o; fi
